diff --git app/controllers/photos_controller.rb app/controllers/photos_controller.rb
index 1f8a371..0eebfb9 100644
--- app/controllers/photos_controller.rb
+++ app/controllers/photos_controller.rb
@@ -114,7 +114,7 @@ class PhotosController < ApplicationController
       # get file content type
       att_content_type = (request.content_type.to_s == "") ? "application/octet-stream" : request.content_type.to_s
       # create tempora##l file
-      file = Tempfile.new(file_name, {:encoding =>  'BINARY'})
+      file = Tempfile.new(file_name, '/data/diaspora/tmp', {:encoding =>  'BINARY'})
       # put data into this file from raw post request
       file.print request.raw_post.force_encoding('BINARY')
 
