--- config/environments/production.rb
+++ config/environments/production.rb
@@ -85,8 +85,8 @@ Rails.application.configure do
   # config.log_formatter = ::Logger::Formatter.new

   # Use a different logger for distributed setups.
-  # require 'syslog/logger'
-  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
+  require 'syslog/logger'
+  config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'diaspora')

   # if ENV["RAILS_LOG_TO_STDOUT"].present?   #  logger           = ActiveSupport::Logger.new(STDOUT)