diff --git app/uploaders/unprocessed_image.rb app/uploaders/unprocessed_image.rb
index b662f3b..e0547ef 100644
--- app/uploaders/unprocessed_image.rb
+++ app/uploaders/unprocessed_image.rb
@@ -7,6 +7,14 @@
 class UnprocessedImage < CarrierWave::Uploader::Base
   include CarrierWave::MiniMagick

+  def store_dir
+    '/data/diaspora/uploads'
+  end
+
+  def cache_dir
+    '/data/diaspora/cache'
+  end
+
   attr_accessor :strip_exif

   def strip_exif
